engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
```

Comment:
Previously,  specular power of 1 made specular light do this:
![terasology-160408105317-1280x720](https://cloud.githubusercontent.com/assets/5922109/14395970/c3f22be6-fd89-11e5-9a55-43398f14d47c.jpg)

With the change of the specular glsl shared function,  and with bumping up this value by default,  we now get specular light more like this:
![terasology-160408104659-1280x720](https://cloud.githubusercontent.com/assets/5922109/14395990/e360ea8a-fd89-11e5-8eec-e05f73028c96.jpg)